src/sgame/sg_active.cpp

```diff
@@ -583,7 +583,6 @@ static void SpectatorThink( gentity_t *ent, usercmd_t *ucmd )
 		client->pers.classSelection = PCL_NONE;
 		client->pers.humanItemSelection = WP_NONE;
 		client->ps.stats[ STAT_CLASS ] = PCL_NONE;
-		client->ps.pm_flags &= ~PMF_QUEUED;
```

Comment:
I mean, I can keep it if you want, but it certainly seems duplicated.

Reply:
Well, duplication is not immediately obvious since it's inside of if statements. If you're sure that it works then that's fine

Reply:
I've dropped the commit for now because this is not very important compared to the rest and I don't really have the courage to check this this evening.